draft-ietf-httpbis-binary-message.md

```diff
@@ -0,0 +1,634 @@
+---
+title: "Binary Representation of HTTP Messages"
+abbrev: Binary HTTP Messages
+docname: draft-ietf-httpbis-binary-message-latest
+category: std
+ipr: trust200902
+area: ART
+workgroup: HTTPBIS
```

Comment:
Just 'HTTP', not the short name

Reply:
I thought that we had to use the short name here.  I guess that this is display, not metadata.